docs/source/advanced/test-builders.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Using models in tests'
+---
+
+Apollo Android generates model classes based on your schema and operations. You can then use these classes in your
+project, which means you will probably need to instantiate them in your tests.
+
+However, if your models have many fields or their hierarchy is deep, you may find it tedious to instantiate them
+manually - as a value must be set for each field.
+
+To make it easier, Apollo Android provides test builders: an optional mechanism that helps to build these models, with
+default values.
```

Comment:
```suggestion
To make it easier, Apollo Android provides Test Builders: an optional mechanism that helps to build these models, with default values.
Additionally, Test Builders will populate `__typename` for you and deduplicate merged fields when possible.
```

Reply:
Not 100% sure about `test builders` vs `Test Builders`. Uppercase somewhat "promotes" them but maybe `test builders` is better. @StephenBarlow do we have any guidelines about this?

Reply:
Accepting your suggest change for now (including Test Builders) but open to keep it lower caps too 😊  I like Test Builders because I find it clarifies a bit that we're talking a bout a specific feature.